source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -627,6 +627,7 @@ def __init__(self,windowHandle=None,IAccessibleObject=None,IAccessibleChildID=No
 		self.event_objectID=event_objectID
 		self.event_childID=event_childID
 		super(IAccessible,self).__init__(windowHandle=windowHandle)
+		self.uniqueID = (self.windowHandle, self.IA2UniqueID)

```

Comment:
It would be better if this was a lazy property rather than set in the constructor.
The Generic IAccessible code I'm talking about can also go here.
E.g.:
```
def _get_uniqueID(self):
	if self.ia2UniqueID:
		return (self.windowHandle,self.ia2UniqueID)
	elif self.IAccessibleIdentity:
		return self.IAccessibleIdentity
	else:
		return (self.windowHandle,hash(self.IAccessibleObject)
```